### ISSUE TITLE
AMP Preview button not working - just triggering a reload

### ISSUE THREAD

**Issue by braican** [Automattic/amp-wp#1597]
We have a site that is having an issue with the AMP preview button not working as expected. When the AMP preview button is clicked, the page triggers a reload rather than opening a new tab to the preview.

If no new content has been added, the admin page reloads. If new content _has_ been added, the `Leave Site` Javascript alert pops up:

![screen shot 2018-11-08 at 1 30 33 pm](https://user-images.githubusercontent.com/3286676/48219458-8e5fb980-e35a-11e8-87c4-7565aa724328.png)

We're using version `0.7.2` of the AMP plugin, on WordPress version `4.9.8`. The site is hosted on WP Engine.

Some additional context that may be useful - the AMP preview button _does_ work on our staging WP Engine site...this issue is only occurring on production for us.

Happy to provide any other bit of information that may be useful.

**Comment by westonruter** [comment]
If you open your DevTools console, is there any JavaScript error that is being raised?

**Comment by westonruter** [comment]
Do you have any different plugins active on staging vs production?

**Comment by braican** [comment]
OK, I had missed this the first time. There were different versions of ACF Pro on our staging vs. our production environments. Staging was using ACF Pro v5.5.12, where production was using the latest ACF Pro v5.7.7.

Testing locally, I was able to identify that the AMP preview button works with ACF Pro up to version 5.7.4. Testing with ACF Pro version 5.7.5 broke the AMP preview button. It appears that something changed in ACF Pro version 5.7.5 is the culprit.

**Comment by westonruter** [comment]
Is that something you can raise with ACF support?

**Comment by westonruter** [comment]
Thanks. Please feel free to continue commenting after I close this. We can re-open if needed.

**Comment by ethanclevenger91** [comment]
@braican Can you link to your support ticket, if it's public? Interested in the status on this.

**Comment by braican** [comment]
Unfortunately the support ticket is behind a private account, but the last update I got on the ticket was from Elliot Condon, the plugin author:

> It is most likely our validation or unload manager which is causing the conflict.
I'll add this to my to-do list for the day and will be in touch with some solution shortly.

This was back in November though, and I never got any other update (though the ticket is now closed). They've released a few versions since PRO 5.7.7 so it's possible they addressed it in one of the subsequent versions, though I don't see mention of an AMP bug in any of the release notes.

For us, rolling back to the 5.7.4 version was a fine fix and I haven't really kept up with this since originally looking into it.
